### DISCUSSION TITLE
能否开源训练数据

### DISCUSSION THREAD

**Discussion by shudct** [PaddlePaddle/PaddleOCR#12760] [Q&A]
表格识别任务，查看预训练模型发现训练数据来源三个数据集：
![截屏2024-06-06 18 04 30](https://github.com/PaddlePaddle/PaddleOCR/assets/11989304/f0f87d90-d80f-4301-a14a-ef45f05aec33)
能否开源这部分数据用于表格识别做进一步的优化

**Comment by SWHL** [comment]
这个开源不太可能，不过可以自己生成啊，用这个[TableGeneration](https://github.com/WenmuZhou/TableGeneration/tree/main)

**Reply by shudct** [reply]
伪造的数据训练后在真实数据上评估质量很差的

**Comment by GreatV** [comment]
pubtabnet 和 wtw 都是开源的数据集

**Reply by shudct** [reply]
pubtabnet部分cell的坐标是文字，hwl的是单元格，这部分，你们有做什么特别处理吗？

WTW标注的是xml，github自带的xmltohtml脚本有问题，想问下你们是怎么处理这部分数据的？
